src/actions/websocket.js

```diff
@@ -123,6 +126,8 @@ async function handleReconnect(dispatch, getState) {
     const {currentChannelId} = entities.channels;
     const {currentUserId} = entities.users;
 
+    getLicenseConfig()(dispatch, getState);
```

Comment:
1/5 on this one minor thing, this can be written as `dispatch(getLicenseConfig())` the other ones as well

Reply:
Doing it that way also prevents us from needing to care if the action is a thunk action (that returns a function) or a regular redux action (that returns a JS object).